<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>domain8</class>
 <widget class="QDialog" name="domain8">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>391</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>preamble: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEditPreamble"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>id: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEditId"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>type: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEditType"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>B2: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEditB2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>B1: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEditB1"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>B0: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEditB0"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnConfirm8">
      <property name="text">
       <string>чбошод</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label</zorder>
  <zorder>lEditPreamble</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>btnConfirm8</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>layoutWidget_5</zorder>
  <zorder>lEditB2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
